packages/venia-ui/lib/queries/getNavigationMenu.graphql

```diff
@@ -13,6 +13,7 @@ query navigationMenu($id: Int!) {
                     small_image {
                         url
                     }
+                    url_key
```

Comment:
AFAICT this whole sub-query isn't used anywhere so we can just remove it entirely.

Reply:
Done: https://github.com/magento/pwa-studio/pull/1935/commits/bf4b0dd741eddc8ed586fffbea6245814fd7b94d